src/colorizer/CanvasOverlay.ts

```diff
@@ -300,16 +300,16 @@ export default class CanvasOverlay implements IRenderCanvas {
 
     // If the dataset has changed types, construct and initialize the inner
     // canvas.
-    let hasUpdatedCanvasParams = false;
+    let hasAlreadyUpdatedCanvasParams = false;
```

Comment:
Renamed this because I found myself getting confused by what it meant at a glance